plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -39,36 +39,14 @@ protected function get_block_type_uses_context() {
 	}
 
 	/**
-	 * Get stock text based on stock. For example:
-	 * - In stock
-	 * - Out of stock
-	 * - Available on backorder
-	 * - 2 left in stock
+	 * Get the product types that are allowed in the editor.
 	 *
-	 * @param bool     $is_in_stock Whether the product is in stock.
-	 * @param bool     $is_low_stock Whether the product is low in stock.
-	 * @param int|null $low_stock_amount The amount of stock that is considered low.
-	 * @param bool     $is_on_backorder Whether the product is on backorder.
-	 * @return string Stock text.
+	 * @return array
 	 */
-	protected static function getTextBasedOnStock( $is_in_stock, $is_low_stock, $low_stock_amount, $is_on_backorder ) {
-		if ( $is_low_stock ) {
-			return sprintf(
-				/* translators: %d is number of items in stock for product */
-				__( '%d left in stock', 'woocommerce' ),
-				$low_stock_amount
-			);
-		} elseif ( $is_on_backorder ) {
-			return __( 'Available on backorder', 'woocommerce' );
-		} elseif ( $is_in_stock ) {
-			return __( 'In stock', 'woocommerce' );
-		} else {
-			return __( 'Out of stock', 'woocommerce' );
-		}
+	protected function product_types_without_stock_indicator() {
```

Comment:
Nit: Personal preference, but for functions and methods, I prefer it they start with a verb, so in this case this could be `get_product_types_without_stock_indicator()`. Alternatively, this could be converted to a property of the PHP class.